src/shared/qrcode-util.c

```diff
@@ -48,6 +48,25 @@ static void print_border(FILE *output, unsigned width) {
         }
 }
 
+static void print_positioned_border(FILE *output, unsigned width, unsigned row, unsigned column) {
```

Comment:
Hmm. `_positined` version of the functions should be merged with the original ones.
Instead of introducing new functions, 
1. please make `print_border()` and `print_qrcode()` take `row` and `column` (if they are `UINT_MAX` ignore the values),
2. rename `print_qrcode()` -> `print_qrcode_full()`,
3. then introduce an inline function in the header that takes `UINT_MAX` for `row` and `column`.